### PR TITLE
Translation of performance chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

### CODE REVIEW

File: src/content/ja/2021/performance.md [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+  )
+}}
+
+最後に、<a hreflang="en" href="https://datastudio.google.com/s/o6zLzlTpWaI">Core Web Vitalsテクノロジーレポート</a> を見ると、各CMSが指標別にどのようなパフォーマンスを見せてくれるかがわかります。
+
+{{ figure_markup(
+  link="https://datastudio.google.com/s/o6zLzlTpWaI",
+  image="cms_ttfb_cwv_report.png",
+  caption='CMSによる良いTTFBを持つオリジン (<a hreflang="en" href="https://datastudio.google.com/s/o6zLzlTpWaI">Core Web Vitalsテクノロジーレポート</a>)',
+  description="WordPress、Wix、Squarespaceの「Origin Having Good TTFB」の時系列チャート。7月は、WordPressとWixが5％、Squarespaceが20％でした。"
```
Comment (tunetheweb) [inline src/content/ja/2021/performance.md:222]: Should "Origin Having Good TTFB" be translated?

File: src/content/ja/2021/performance.md [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+featured_stat_label_1: 上位1,000サイトのうちCore Web Vitalsを通過したサイトの割合
+featured_stat_2: 79%
+featured_stat_label_2: デスクトップで画像をLCP要素として表示したページ
+featured_stat_3: 67 s
+featured_stat_label_3: 最長の総ブロッキング時間
+---
+
+## 序章
+
+ユーザー体験にとって、パフォーマンスは重要です。読み込みが遅く、反応が遅いWebサイトは、ユーザーをイライラさせ、コンバージョンの低下を招きます。<a hreflang="en" href="https://web.dev/vitals/">Core Web Vitals</a>が<a hreflang="en" href="https://developers.google.com/search/blog/2020/11/timing-for-page-experience">Google検索ランキング</a>に寄与するのは今年がはじめてです。そのため、ウェブサイトのパフォーマンス向上に対する関心が高まっており、ユーザーにとっては嬉しいニュースです。
```
Comment (tunetheweb) [inline src/content/ja/2021/performance.md:23]: web.dev has started to become avaialble in Japanese!

```suggestion
ユーザー体験にとって、パフォーマンスは重要です。読み込みが遅く、反応が遅いWebサイトは、ユーザーをイライラさせ、コンバージョンの低下を招きます。<a hreflang="ja" href="https://web.dev/i18n/ja/vitals/">Core Web Vitals</a>が<a hreflang="en" href="https://developers.google.com/search/blog/2020/11/timing-for-page-experience">Google検索ランキング</a>に寄与するのは今年がはじめてです。そのため、ウェブサイトのパフォーマンス向上に対する関心が高まっており、ユーザーにとっては嬉しいニュースです。
```

Could you check the rest of these web.dev links?

**Review by tunetheweb — CHANGES_REQUESTED** [review]
Looks good!

One question, and a request to update web.dev links.

File: src/content/ja/2021/performance.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2021/performance.md:222]: Thanks, I translated.

File: src/content/ja/2021/performance.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2021/performance.md:23]: I didn't know.
Thanks, I translated.

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**